tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -381,8 +381,9 @@ def test_multiple_code_mapping_matches(self):
             response = self.get_success_response(
                 self.organization.slug, self.project.slug, qs_params={"file": self.filepath}
             )
-            # Assert that the code mapping that is user generated is chosen
-            assert response.data["config"] == self.expected_configurations(self.code_mapping1)
+            # Assert that the code mapping that is user generated and has the most defined stack
+            # trace of the user generated code mappings is chosen
+            assert response.data["config"] == self.expected_configurations(self.code_mapping3)
```

Comment:
Check that the best code mapping was chosen now that stack root precedence is added